course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -105,12 +105,24 @@
                 });
             };
 
+            $scope.getCourseDescription = function(course) {
+                // If a course's title is [NULL], attempt to display the short title.
+                // If the short title is also [NULL], display [School] 'Untitled Course' [Term Display]
+                if(typeof course.title != "undefined" && course.title.length > 0){
+                    return course.title;
+                }
+                else if(typeof course.short_title != "undefined" && course.short_title.length > 0){
+                    return course.short_title;
+                }
+                return 'Untitled Course';
+            };
+
             $scope.courseInstanceToTable = function(course) {
                 // This logic is reused in PeopelController.getFormattedCourseInstance
                 // So any changes to data format need to be done in both places.
                 // TODO: Eventually move the reusable logic to a separate library
                 var cinfo = {};
-                cinfo['description'] = course.title;
+                cinfo['description'] = $scope.getCourseDescription(course);
```

Comment:
As per the method docstring above, don't we need to update the PeopleController as well?